plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestingPostgreSqlServer.java

```diff
@@ -82,13 +83,13 @@ private static void execute(String url, Properties properties, String sql)
         Iterator<String> logsIterator = logs.iterator();
         ImmutableList.Builder<RemoteDatabaseEvent> events = ImmutableList.builder();
         while (logsIterator.hasNext()) {
-            String logLine = logsIterator.next();
+            String logLine = logsIterator.next().replaceAll(LOG_PREFIX_REGEXP, "");
```

Comment:
I think timestamps are useful sometimes on CI - for example when a certain test appears slow or stuck.

Reply:
Agree, and that's why they added to the log in Postgres 10+